### ISSUE TITLE
Instrument Measured Variables for a deployment and associated results

### ISSUE THREAD

**Issue by AmberSJones** [UCHIC/ODM2Sensor#232]
It’s unclear to my why the Instrument Measured Variables table is blank here. When a result is created, it should populate the instrument measured variables for that deployment - or this table should already be populated with the Instrument Measured Variables that can be measured by that instrument. 

![image](https://user-images.githubusercontent.com/5023186/34800415-d4c2f55e-f620-11e7-9569-91f2b52ac2fa.png)

**Comment by Elijahwalkerwest** [comment]
PLease provide specific deployment ID. Every deployment I've checked so far that table is populated correctly.

**Comment by AmberSJones** [comment]
Here is the link to the screenshot that I posted. I haven't made a thorough search.
http://django.uwrl.usu.edu/equipment/actions/deployment-detail/524/

**Comment by AmberSJones** [comment]
These may just not be populated. Please check @Elijahwalkerwest . Here's another few - all from the same site. 
http://django.uwrl.usu.edu/equipment/actions/deployment-detail/523/
http://django.uwrl.usu.edu/equipment/actions/deployment-detail/525/

**Comment by Elijahwalkerwest** [comment]
After looking in to this, there's nothing broken here. Those equipment models simply don't have any Output variables created for them yet. I created a test one for 
http://django.uwrl.usu.edu/equipment/actions/deployment-detail/524/

However it seems there is no way to deleted an instrumentoutputvariable, nor does the model details page show variables for a given model. 

I assume these are features that should be present?

**Comment by AmberSJones** [comment]
Yes, let's add both the view of InstrumentOutputVariables on the EquipmentModel page as well as the ability to add and delete InstrumentOutputVariable from the EquipmentModel page.

**Comment by Elijahwalkerwest** [comment]
This is implemented. Right now however you need to go to the details of an IOV to delete it. Is that behavior acceptable? Or would you like it another way?

**Comment by AmberSJones** [comment]
I don't see where this appears on the Equipment Model page?
